mocca-client/src/main/java/com/paypal/mocca/client/MoccaFeignEncoder.java

```diff
@@ -135,6 +138,14 @@ static SelectionSet getSelectionSet(RequestTemplate requestTemplate) {
         Parameter[] parametersMetadata = requestTemplate.methodMetadata().method().getParameters();
         List<Variable> variables = new ArrayList<>(parameters.length);
         for (int i = 0; i < parameters.length; i++) {
+
+            // TODO We could make it configurable, leaving for the user to decide if null variables should
```

Comment:
Is this worthy of an issue?

Reply:
I am not sure. Let's wait and see if users ask for it.